app/src/main/java/com/example/sharingang/MainActivity.kt

```diff
@@ -73,6 +74,12 @@ class MainActivity : AppCompatActivity(R.layout.activity_main) {
         setupShakeListener()
 
         handleDeepLink()
+
+        createChannel(
+            this.getString(R.string.new_item_notification_channel_id),
+            this.getString(R.string.new_item_notification_channel_name),
```

Comment:
I think that you should define these constants in a Kotlin file instead because strings.xml is used for translations

Reply:
Yeah probably, I followed the documentation and Kotlin tutorial on that one to be honest.

Reply:
You can keep it like that then. I don't know what are the best practices for it